test.sh

```diff
@@ -3,6 +3,8 @@
 # Fail early
 set -eu -o pipefail
 
+NGROK_URL="https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip"
```

Comment:
That's where the [ngrok Download](https://ngrok.com/download) page points. [Equinox.io](https://equinox.io/) seems to be some sort of Go project hosting.